org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -333,52 +343,63 @@ public static Location toLocation(ICompilationUnit unit, int offset, int length)
 	 * @return location or null
 	 * @throws JavaModelException
 	 */
-	public static Location toLocation(IClassFile unit, int offset, int length) throws JavaModelException{
-		Location result = new Location();
-		String packageName = unit.getParent().getElementName();
-		String jarName = unit.getParent().getParent().getElementName();
+	public static Location toLocation(IClassFile classFile, int offset, int length) throws JavaModelException{
+		String packageName = classFile.getParent().getElementName();

```

Comment:
Haven't seen any issues with default package so far, even though getParent could be null in theory